bentoml/yatai/client/bento_repository_api.py

```diff
@@ -195,12 +205,29 @@ def upload_from_dir(self, saved_bento_path, labels=None):
             )
 
         if response.uri.type == BentoUri.LOCAL:
-            if os.path.exists(response.uri.uri):
-                # due to copytree dst must not already exist
-                shutil.rmtree(response.uri.uri)
-            shutil.copytree(saved_bento_path, response.uri.uri)
+            # For remote yatai service, call upload method.
```

Comment:
nit:

# When using Yatai backed by File System repository, if Yatai is a local instance, copy the files directly.
# Otherwise, use UploadBento RPC to stream files to remote Yatai server